cypress/cypress/component/Input/BasicInputField.cy.tsx

```diff
@@ -52,6 +52,30 @@ describe("Input.cy.tsx", () => {
     cy.get("@onSubmitSpy").should("be.calledOnceWith", { [name]: randomNumber });
   });
 
+  it("number accepts step", () => {
```

Comment:
I would call the test "number input correctly increase value by step"

Reply:
@manni497 test is renamed